lib/strategy/EnrichClientFromRemoteConfigStrategy.js

```diff
@@ -44,35 +47,53 @@ EnrichClientFromRemoteConfigStrategy.prototype._resolveApplicationByName = funct
   });
 };
 
-EnrichClientFromRemoteConfigStrategy.prototype.process = function (config, callback) {
-  var application = config.application;
+// If there are only two applications and one of them is
```

Comment:
What if there's only one application, and its name is `Stormpath`? Wouldn't it be nice to use it as default?